package.json

```diff
@@ -49,8 +50,18 @@
     "tape": "~4.0.0"
   },
   "scripts": {
-    "test": "tape test/*-test.js | faucet"
+    "install": "node-pre-gyp install --fallback-to-build",
+    "test": "tape test/*-test.js | faucet",
+    "prebuild": "node-pre-gyp rebuild && node-pre-gyp package"
   },
   "license": "MIT",
-  "gypfile": true
+  "gypfile": true,
+  "bundledDependencies": [
```

Comment:
Fwiw, this seems to work fine https://github.com/Level/leveldown/blob/node-pre-gyp-and-travis/package.json#L30-L36 with travis